packages/features/ee/organizations/lib/orgDomains.ts

```diff
@@ -60,5 +62,5 @@ export function getSlugOrRequestedSlug(slug: string) {
         },
       },
     ],
-  };
+  } satisfies Prisma.TeamWhereInput;
```

Comment:
@hariombalhara  UI build is failing because of this 

![image](https://github.com/calcom/cal.com/assets/53316345/25646e70-a05d-4b3e-a032-43290c2f665e)